src/data/quizzes/securityx-quiz.ts

```diff
@@ -688,6 +692,22 @@ const securityXQuiz = {
         {
           title: 'Exploit Frameworks',
           url: 'https://www.prosec-networks.com/en/blog/exploit-frameworks/'
+        },
+        {
+          title: 'What is Software Composition Analysis?',
+          url: 'https://www.paloaltonetworks.com/cyberpedia/what-is-sca'
+        },
+        {
+          title: 'What is Reverse Engineering Technique in Cybersecurity?',
+          url: 'https://www.geeksforgeeks.org/computer-networks/what-is-reverse-engineering-technique-in-cybersecurity/'
+        },
+        {
+          tite: 'Identifying Vulnerable Code',
```

Comment:
You have a typo causing the build to fail.
```suggestion
          title: 'Identifying Vulnerable Code',
```

Reply:
Fixed this issue.